extensions/stackdriver/stackdriver.cc

```diff
@@ -265,6 +268,26 @@ bool StackdriverRootContext::configure(size_t configuration_size) {
   }
 
   if (!logger_ && enableAccessLog()) {
+    std::unordered_map<std::string, std::string> extra_labels;
+    if (config_.has_custom_log_config()) {
+      cleanupExpressions();
+      for (const auto& dimension : config_.custom_log_config().dimensions()) {
+        uint32_t token;
+        if (createExpression(dimension.second, &token) != WasmResult::Ok) {
```

Comment:
This is correct: we should create expressions at config time only. Can you add deleteExpression in cleanup to release host resources? Otherwise, we're leaking some data structures.

Reply:
Thanks for catching. Fixed